client/src/app/_helpers/errors.400.ts

```diff
@@ -1,18 +1,18 @@
 export const ERRORS400 = {
-    'LoginError': 'Erreur d\'authentification: combinaison username / mot de passe invalide.',
-    'LogoutError': 'Erreur de deconnexion: la déconnexion a échouée du côté serveur.',
-    'RemovedValueProhibited': 'Suppression impossible, la requête n\'autorise pas la suppression d\'associations.',
-    'BadRequest': 'Requête mal formée: la requête ne respecte pas le format imposé par le serveur.',
-    'UnknownUser': 'Utilisateur inconnu: l\'utilisateur est introuvable dans la base de données.',
-    'UnknownMember': 'Adhérent inconnu: l\'adhérent est introuvable dans la base de données.',
-    'UnknownKommission': 'Kommission inconnue: la kommission est introuvable dans la base de données.',
-    'UnknownBarman': 'Barman inconnu: Le barman est introuvable dans la base de données.',
-    'UnknownService': 'Service inconnu: le service est introuvable dans la base de données.',
-    'UnverifiedUsername': 'Username non verifié: le username (email) n\'a pas été vérifié, merci de vérifier vos e-mails.',
-    'UndefinedPassword': 'Le mot de passe n\'a pas été défini, merci de vérifier vos e-mails.',
-    'UnknowPasswordToken': 'Token inconnu: le \'PasswordToken\' est inconnu de la base de données.',
-    'UnknowUsernameToken': 'Token inconnu: le \'UsernameToken\' est inconnu de la base de données.',
-    'MailerError': 'L\'envoi du mail du côté serveur a échoué.',
-    'CodeError': 'Code invalide: Le code n\'est pas valide.',
-    'BadUsername': 'L\'username n\'est pas disponible',
+    LoginError: 'Erreur d\'authentification: combinaison username / mot de passe invalide.',
+    LogoutError: 'Erreur de deconnexion: la déconnexion a échouée du côté serveur.',
+    RemovedValueProhibited: 'Suppression impossible, la requête n\'autorise pas la suppression d\'associations.',
+    BadRequest: 'Requête mal formée: la requête ne respecte pas le format imposé par le serveur.',
+    UnknownUser: 'Utilisateur inconnu: l\'utilisateur est introuvable dans la base de données.',
+    UnknownMember: 'Adhérent inconnu: l\'adhérent est introuvable dans la base de données.',
+    UnknownKommission: 'Kommission inconnue: la kommission est introuvable dans la base de données.',
+    UnknownBarman: 'Barman inconnu: Le barman est introuvable dans la base de données.',
+    UnknownService: 'Service inconnu: le service est introuvable dans la base de données.',
+    UnverifiedUsername: 'Username non verifié: le username (email) n\'a pas été vérifié, merci de vérifier vos e-mails.',
+    UndefinedPassword: 'Le mot de passe n\'a pas été défini, merci de vérifier vos e-mails.',
+    UnknowPasswordToken: 'Token inconnu: le \'PasswordToken\' est inconnu de la base de données.',
+    UnknowUsernameToken: 'Token inconnu: le \'UsernameToken\' est inconnu de la base de données.',
+    MailerError: 'L\'envoi du mail du côté serveur a échoué.',
+    CodeError: 'Code invalide: Le code n\'est pas valide.',
+    BadUsername: 'L\'username n\'est pas disponible'
```

Comment:
In back we have defined a mandatory comma at the end of object like that, could we follow here too?